samples/VanillaJSTestApp/JavascriptSPA/authConfig.js

```diff
@@ -1,7 +1,7 @@
 // Config object to be passed to Msal on creation
 const msalConfig = {
     auth: {
-        clientId: "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
+        clientId: "6226576d-37e9-49eb-b201-ec1eeb0029b6",
```

Comment:
Why did this change?

Reply:
The other client id is not configured for non Microsoft accounts , I needed to test against msa as well

Reply:
@pkanher617 I think changed this for testing, interested what he thinks.